source/_docs/z-wave/controllers.markdown

```diff
@@ -63,6 +63,19 @@ The alternative to a stick is a hub that supports Z-Wave. Home Assistant support
 
 ## Controller Notes
 
+### 800 Series Controllers
+
+Some features are not supported with 800 series controllers yet.
+Unsupported
+ - Long Range
+ - NVM Backup/Restore 
+Supported
+ - Device inclusion + exclusion (Basic and SmartStart)
+ - Setting Device Configuration Values
+ - Getting Device Configuration Values
+ - Command Classes: Multilevel Switch v2 and v4, Color Switch v1, and Config v1 and v2
```

Comment:
This seems like an arbitrary list of supported features and sounds like only these CCs and versions are supported, when the list is much longer in reality.
Aside from that, sending specific commands is not a controller-specific feature.